src/common/hooks/useCurrentSettingsLevel.ts

```diff
@@ -0,0 +1,31 @@
+/**
+ * Invoice Ninja (https://invoiceninja.com).
+ *
+ * @link https://github.com/invoiceninja/invoiceninja source repository
+ *
+ * @copyright Copyright (c) 2022. Invoice Ninja LLC (https://invoiceninja.com)
+ *
+ * @license https://www.elastic.co/licensing/elastic-license
+ */
+
+import { SettingsLevel } from '../enums/settings';
+import { useActiveSettingsDetails } from './useActiveSettingsDetails';
+
+export const useCurrentSettingsLevel = () => {
```

Comment:
Please follow the conventions of the project and use `function $name`.

Reply:
Changed!